contracts/contracts/util.cairo

```diff
@@ -0,0 +1,44 @@
+from starkware.cairo.common.math import assert_not_zero
```

Comment:
I think this file should have its own unit test file

Reply:
Did you forget to commit "utils" file and the new test file?

Reply:
Added them